relation_engine_server/test/spec_release/README.md

```diff
@@ -1,8 +1,22 @@
+## Test Spec Release
 
-The file in this directory, `spec.tar.gz` is a cached release of the `relation_engine_spec` repo, found here:
+`sample_spec_release`, and the corresponding archive, `spec.tar.gz`, contain a set of sample schema files suitable for use in tests.
```

Comment:
Seems like we should remove `spec.tar.gz`? That was only there before because the repos were separated and served as a cache of the specs that we don't have to fetch over the network, which seems moot now

Reply:
I'd keep it, as it is exercising the `pull_spec.py` code, which would not be run otherwise; ideally it could be used to create some tests for `pull_spec.py`, too, since we don't have anything that checks the archive expansion/loading explicitly.

Reply:
Ok, since it is just used to test the pull_spec module, then it might be a good update to automate its creation in those tests. Python has a builtin `tarfile` lib that supports gzipping, so it should only be a few lines in the test. We can do it in a future update.